etl/scripts/countries-regions/06-add-continents.ipynb

```diff
@@ -13,7 +13,7 @@
   },
```

Comment:
Are there any actual changes to this notebook or were you just checking it out and then it got cleaned with `jupyter-notebook-cleanup`? When I compare scripts vs notebooks, a big advantage of notebooks is committing outputs into the repository (where I can easily check what does it do without having to rerun it myself). Without those outputs, I'd rather use python scripts (with the advantage of being easily reviewable and more straightforward to debug). What do you think about making `jupyter-notebook-cleanup` optional?

Reply:
I removed the code `OWID_KOR` for Korea since there is another one for "United Korea" that we want to prefer to use instead.